src/app/models/quiz.rb

```diff
@@ -1,6 +1,7 @@
 class Quiz < ApplicationRecord
   validates :title, presence: true
+  has_many :questions
```

Comment:
Why were these changes made? The lambda prevented blank questions from existing, and the `dependent => destroy` said that if the quiz is deleted, delete its questions as well.